src/sonic-config-engine/minigraph.py

```diff
@@ -573,7 +574,9 @@ def parse_meta(meta, hname):
                     region = value
                 elif name == "CloudType":
                     cloudtype = value
-    return syslog_servers, dhcp_servers, ntp_servers, tacacs_servers, mgmt_routes, erspan_dst, deployment_id, region, cloudtype
+                elif name == "SynchronousMode":
```

Comment:
>SynchronousMode [](start = 30, length = 15)

The input minigraph.xml has no new field for you to parse.

Reply:
Removed the field parser in minigraph .xml